community/en/BlackScholes_tf_pricing.ipynb

```diff
@@ -1,25 +1,39 @@
 {
```

Comment:
<p>Please put the second pip install on a different line.</p><p><br></p><p>```</p><p>!pip install --upgrade tf-nightly-2.0-preview </p><p>!pip install --upgrade tfp-nightly</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/examples/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Done! Thanks.